rusk-wallet/src/bin/interactive.rs

```diff
@@ -718,6 +736,18 @@ fn confirm(cmd: &Command) -> anyhow::Result<bool> {
             println!("   > Max fee = {} DUSK", Dusk::from(max_fee));
             prompt::ask_confirm()
         }
+        Command::MoonlightUnstake {
```

Comment:
same as above, and several others, I think these commands should be added in a relevant PR

Reply:
@miloszm Removed